user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserQuotasOperator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.user.operator;
+
+import io.strimzi.operator.common.operator.resource.ReconcileResult;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+public class KafkaUserQuotasOperator {
```

Comment:
I wonder whether we really need this class _as well as_ `KafkaUserQuotas`. Can't you put all the code in the one class?

Reply:
You mean merging `KafkaUserQuotasOperator` and `KafkaUserQuotas`?

Reply:
Yes